controllers/funding/materials.js

```diff
@@ -191,7 +217,9 @@ function initForm({ router, routeConfig }) {
                                 sendTo: customerSendTo,
                                 subject: 'Thank you for your Big Lottery Fund order',
                                 templateName: 'emails/newMaterialOrder',
-                                templateData: {}
+                                templateData: {
+                                    locale: req.i18n.getLocale()
```

Comment:
Maybe add a TODO here to try and work out why we need this?

Reply:
Done